handlers/workloads.go

```diff
@@ -68,7 +70,23 @@ func WorkloadList(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if criteria.IncludeHealth {
-		rateInterval, err := adjustRateInterval(r.Context(), businessLayer, p.Namespace, p.RateInterval, p.QueryTime)
+		// When the cluster is not specified, we need to get it. If there are more than one,
+		// get the one for which the namespace creation time is oldest
+		clusters, _ := businessLayer.Namespace.GetNamespaceClusters(r.Context(), p.Namespace)
+		if len(clusters) == 0 {
+			err = fmt.Errorf("No clusters found for namespace  [%s]", p.Namespace)
+			handleErrorResponse(w, err, "Error looking for cluster: "+err.Error())
+			return
+		}
+		var cluster string
+		var creationTimestamp time.Time
+		for i, cl := range clusters {
+			if i == 0 || cl.CreationTimestamp.Before(creationTimestamp) {
+				cluster = cl.Cluster
+			}
+		}
```

Comment:
Same comment here about `creationTimestamp` and instead using the `GetOldestNamespace` helper func.

Reply:
Updated.